src/Compilers/Core/MSBuildTask/Shared/Microsoft.CSharp.Core.targets

```diff
@@ -54,6 +55,7 @@
       <Prefer32Bit>false</Prefer32Bit>
     </PropertyGroup>
 
+    <!-- TODO: Remove this ItemGroup once it has been moved to "_GenerateCompileInputs" target in Microsoft.Common.CurrentVersion.targets -->
```

Comment:
I'll send a separate VSO PR to move this ItemGroup to Microsoft.Common.CurrentVersion.targets (with an additional line to clear out AdditionalFileItemNames at the end). When that change flows back, we can remove this ItemGroup from C# and VB specific core targets.